server/Html.tsx

```diff
@@ -71,7 +71,7 @@ const Html = (props: Props) => {
 					href="/opensearch.xml"
 				/>
 			</head>
-			<body>
+			<body className={props.initialData.scopeData.elements.activeTargetType}>
```

Comment:
To summarize what I was saying on the dev call — `activeTargetType` will be `community` on Pages and `collection` on Collections, but most users will want to apply styles uniformly to both types of layout. Rather than making them type:
```css
.community .thing, .collection .thing {
    /* and stuff */ 
}
```
I propose a new prop on `<Html>` called `bodyClassPrefix` which if set to e.g. `layout` would yield a `body.layout-body-wrapper` item — wordy, but unambiguously not any of the other classes we have floating around relating to layouts. This should be set to the same value on Pages and Collection layouts, and then something else on Pubs.

I'm all for including `activeTargetType` as e.g. `active-target-pub` as well if that seems like it will be helpful.

Reply:
Thanks, good suggestion. Take a look at the rewritten PR.